apps/sbc/ParamReplacer.cpp

```diff
@@ -118,7 +127,7 @@ string replaceParameters(const string& s,
 			 const AmSipRequest& req,
 			 const SBCCallProfile* call_profile,
 			 const string& app_param,
-			 AmUriParser& ruri_parser, 
+			 AmUriParser& ruri_parser,
```

Comment:
are these whitespace changes necessary? even if it's more beautiful without, they just confuse the (git) history of the files/lines. => please remove only whitespace changes in the patch

Reply:
no they come from the editor, I will try to minimize the confusing by reducing unnecessary changes